org.fcitx.Fcitx5.Addon.M17N.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: org.fcitx.Fcitx5.Addon.M17N
+branch: stable
+runtime: org.fcitx.Fcitx5
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
+build-extension: true
+appstream-compose: false
+separate-locales: false
+build-options:
+  prefix: /app/addons/M17N
+  prepend-pkg-config-path: /app/addons/M17N/lib/pkgconfig:/app/addons/M17N/share/pkgconfig
+cleanup:
+  - /bin
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - '*.la'
+modules:
+  - modules/fmt.yaml
```

Comment:
why a separate manifest?

Reply:
Basically we maintain a repo and flathub manifest are generated from it and we have our own shared modules https://github.com/fcitx/flatpak-fcitx5/tree/master/modules.. I'd prefer to keep it like this.

Reply:
ok